lib/lunchbot_web/router.ex

```diff
@@ -44,6 +44,7 @@ defmodule LunchbotWeb.Router do
     end
 
     live "/list-orders", ListOrdersLive
```

Comment:
Would it be possible to add these routes to the live session up above just to keep all the live paths together?